contributing/TESTING.md

```diff
@@ -223,6 +223,12 @@ To run the tests on Sauce Labs:
    ```
 * It may take a few minutes for the tests to start.  You can see the status of your tests on the Sauce Labs [Automated Tests](https://saucelabs.com/beta/dashboard/tests) dashboard.  (You can also see the status of your proxy on the [Tunnels](https://saucelabs.com/beta/tunnels) dashboard.
 
+## Big Thanks
```

Comment:
Instead of creating a separate section, just add the line below to this section. In other words, just remove the section header in the line above.